modules/machine-config-drift-detection.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="machine-config-drift-detection_{context}"]
+= Understanding configuration drift detection
+
+There might be situations when the on-disk state of a node differs from what is configured in the machine config. This is known as _configuration drift_. For example, a cluster admin might manually modify a file, a systemd unit file, or a file permission that was configured through a machine config. This causes configuration drift. Configuration drift can cause problems between nodes in a Machine Config Pool or when the machine configs are updated.  
+
+The Machine Config Operator (MCO) uses the Machine Config Daemon (MCD) to check nodes for configuration drift on a regular basis. If detected, the MCO sets the node and the Machine Config Pool (MCP) to `Degraded` and reports the error. A degraded node is online and operational. However, it cannot be updated.
+
+The MCD performs configuration drift detection upon each of the following conditions:
+
+* When a node boots.
+* After any of the files (Ignition files and systemd drop-in units) specified in the machine config are modified outside of the machine config.
+* Before a new machine config is applied.
++
+[NOTE]
+====
+If you apply a new machine config to the nodes, the MCD temporarily shuts down configuration drift detection. This shut down is needed because the current machine config necessarily differs from the new machine config. After the new machine config is applied, the MCD restarts detecting configuration drift using the new machine config.
```

Comment:
* This shut down -> This shutdown
* I'm not sure what "necessarily" means in the second sentence. Is this sentence trying to say that the current machine config "might" differ from the new one"? Or that it does differ (in which case maybe "necessarily" isn't needed at all?)

Reply:
@bergerhoffer _This shutdown is needed because this new machine config necessarily differs from the machine config on the nodes._
If you apply a new machine config to nodes, there will be changes, and this new machine config is going to be different than the one currently on the nodes.